phpstan-baseline.neon

```diff
@@ -495,6 +495,11 @@ parameters:
 			count: 1
 			path: src/Handler/CurlFactory.php
 
+		-
+			message: "#^Parameter \\#1 \\$filename of function is_dir expects string, string\\|false given\\.$#"
+			count: 1
+			path: src/Handler/CurlFactory.php
+
```

Comment:
extract this unrelated change to another PR so that it can be merged ASAP?

Reply:
It is not an urgent change. Would be a waste of everyone's time.

Reply:
Well that ended up being included in #2638